AWSIoT/Internal/AWSIoTMQTTClient.m

```diff
@@ -523,18 +536,18 @@ - (void)initWebSocketConnectionForURL:(NSString *)urlString {
         [self.topicListeners removeAllObjects];
     }
     
-    //Setup userName if metrics are enabled
-    NSString *username;
+    //Setup userName if metrics are enabled. We use the connection username as metadata for metrics calculation.
+    NSString *userMetaData;
     if (self.isMetricsEnabled) {
-        username = [NSString stringWithFormat:@"%@%@", @"?SDK=iOS&Version=", SDK_VERSION];
-        AWSDDLogInfo(@"username is : %@", username);
+        userMetaData = [self getUserMetaData];
+        AWSDDLogInfo(@"username is : %@", userMetaData);
```

Comment:
nit: Change "username" in log to "user metadata"

Reply:
Done